Makefile

```diff
@@ -29,21 +29,35 @@ endif
 #
 # Included definitions
 #
-include ./tools/mk/Makefile.defs
-include ./tools/mk/Makefile.node_modules.defs
+
+# triton-mockcloud is not a public module, so override
+ENGBLD_DEST_OUT_PATH ?= /stor/builds
+
+ENGBLD_USE_BUILDIMAGE	= true
+ENGBLD_REQUIRE		:= $(shell git submodule update --init deps/eng)
+include ./deps/eng/tools/mk/Makefile.defs
+TOP ?= $(error Unable to access eng.git submodule Makefiles.)
+
+include ./deps/eng/tools/mk/Makefile.node_modules.defs
 ifeq ($(shell uname -s),SunOS)
-	include ./tools/mk/Makefile.node_prebuilt.defs
+	include ./deps/eng/tools/mk/Makefile.node_prebuilt.defs
+	include ./deps/eng/tools/mk/Makefile.agent_prebuilt.defs
 else
 	NODE_EXEC := $(shell which node)
 	NODE = node
 	NPM_EXEC := $(shell which npm)
 	NPM = npm
 endif
-include ./tools/mk/Makefile.smf.defs
+include ./deps/eng/tools/mk/Makefile.smf.defs
 
-RELEASE_TARBALL = $(NAME)-pkg-$(STAMP).tar.bz2
-RELSTAGEDIR := /tmp/$(STAMP)
+RELEASE_TARBALL = $(NAME)-pkg-$(STAMP).tar.gz
+RELSTAGEDIR := /tmp/$(NAME)-$(STAMP)
 
+BASE_IMAGE_UUID = 1ad363ec-3b83-11e8-8521-2f68a4a34d5d
+BUILDIMAGE_NAME = $(NAME)
+BUILDIMAGE_DESC	= SDC MOCK CLOUD
+BUILDIMAGE_PKGSRC = tftp-hpa-5.2
```

Comment:
Really? Current mockcloud MG entry doesn't have any pkgsrc deps.

Reply:
Looks like I missed MG commit 944e5a76046c59a3a6159019d4df41f5c9a86ff2 which
changed the description and pkgsrc packages.

Reply:
Ah, thanks makes sense now.